specification/azsadmin/resource-manager/azurebridge/Microsoft.AzureBridge.Admin/preview/2016-01-01/AzureBridge.json

```diff
@@ -92,11 +92,7 @@
           "$ref": "#/definitions/Display",
           "readOnly": true
         },
-        "isDataAction": {
```

Comment:
@TheOnlyWei  The chenge needs to be reverted only under the examples folder. we need to keep it here